app/main.py

```diff
@@ -1,3 +1,11 @@
-def shop_trip():
-    # write your code here
-    pass
+from app.trip.trip import Trip
+from app.customer.get_customers import get_list_of_customers
+from app.shop.get_shops import get_list_of_shops
+
+
+def shop_trip() -> None:
+    shops = get_list_of_shops()
+    customers = get_list_of_customers()
```

Comment:
You're reading the file twice inside these functions. You can read the info from the file in advance and then pass the file content to the functions as an argument

Reply:
And I'm not sure these functions should be in a separate module. You can include them here, in main